activesupport/lib/active_support/core_ext/object/json.rb

```diff
@@ -239,9 +239,18 @@ def as_json(options = nil)
   end
 end
 
-class IPAddr # :nodoc:
-  def as_json(options = nil)
-    to_s
+unless IPAddr.method_defined?(:as_json, inherit=false)
```

Comment:
This is not a Python keyword arg. :) There is no need for `inherit=` as it assigns an unused local variable named `inherit` for the class, and incidentally, returns `false` for the second argument of `method_defined?`.

Reply:
@gsamokovarov Thanks!
I fixed it.
https://github.com/rails/rails/blob/51304ef7913b38b9f18d92b8b29387bc97b9f233/activesupport/lib/active_support/core_ext/object/json.rb#L242